components/renku_data_services/session/api.spec.yaml

```diff
@@ -535,6 +535,8 @@ components:
       description: A container image
       type: string
       maxLength: 500
+      # NOTE: regex for an image name, optionally with a tag or sha256 specified
```

Comment:
Can we please reference where we got this from in a comment?
is it from https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pulling-manifests?